modules/rhizo_base/manifests/freeswitch.pp

```diff
@@ -121,7 +121,7 @@
     'freeswitch-mod-syslog', 'freeswitch-mod-tone-stream',
     'freeswitch-mod-voicemail', 'freeswitch-mod-voicemail-ivr',
     'freeswitch-mod-xml-cdr', 'freeswitch-mod-cdr-pg-csv',
-    'freeswitch-sysvinit', 'libfreeswitch1',
+    'libfreeswitch1',
```

Comment:
Good catch. Maybe related fact is that I think it was never 100% decided yet that we would move away from runit to control FS. I am in favour of using systemd

Reply:
sgtm. I know systemd is controversial, but despite people's feelings I think most can agree that having one consistent init system on a deployment, regardless of how unix like it is, is much better than having two managing different interrelated parts of the system : )

Reply:
I won't go into here, but this is not about systemd. 
I don't really see runit and systemd as in the same ball park. Maybe I don't want to have runit manage by entire system, nor would I use pm2 or such either. but for keeping something up. it's great. and stood the test of time. 

TL;DR runit does a thing and it does it well.